UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -878,7 +878,7 @@ <h2>Examples</h2>
 					<aside class="example" title="Compact explanations">
 						<ul>
 							<li>ARIA roles included.</li>
-							<li>Closed captions and sign languages for videos.</li>
+							<li>Closed captions and sign language for videos.</li>
```

Comment:
Suggest making this "sign language interpretation".
```suggestion
							<li>Closed captions and sign language interpretation for videos.</li>
```